jbpm-xes/src/test/java/org/jbpm/xes/XESPersistenceBase.java

```diff
@@ -17,8 +17,9 @@
 package org.jbpm.xes;
 
 import java.util.Properties;
-import org.jbpm.persistence.util.PersistenceUtil;
+
 import org.jbpm.test.JbpmJUnitBaseTestCase;
+import org.jbpm.test.persistence.util.PersistenceUtil;
```

Comment:
I guess `jbpm-xes` module then doesn't need to depend on `jbpm-persistence-jpa` test-jar artifact?

Reply:
Nice catch!

Removing jbpm-persistence-jpa dependencies from pom.xml (not needed)

Reply:
Done.